src/DelegatedRole.php

```diff
@@ -122,7 +122,8 @@ public function matchesPath(string $target): bool
             }
             return false;
```

Comment:
Why don't we have this same `return false` in the block for `(isset($this->pathHashPrefixes))`

Can a delegated roll have both `pathHashPrefixes` and `paths`?

Reply:
No, it cannot, which is why we now throw `LogicException` at the end of the method. This is also enforced by validation (already existing in HEAD).